a3fe/run/simulation.py

```diff
@@ -338,19 +338,6 @@ def run(self, runtime: float = 2.5) -> None:
         -------
         None
         """
-        # Need to make sure that runtime is a multiple of the time per cycle
```

Comment:
Please ensure that the somd.cfg file is written out before here, to ensure that the most up-to-date options are written out before the simulation is run.